internal/utils/k8s/cluster_info_getter.go

```diff
@@ -14,7 +14,7 @@ const (
 	gardenerShootNameAttributeName     = "shootName"
```

Comment:
This constant is not used anymore, right?
If yes, please remove it

Reply:
unused constant removed